quantum/rgblight_breathe_table.h

```diff
@@ -1,468 +1,117 @@
-#ifndef RGBLIGHT_EFFECT_BREATHE_TABLE
+#pragma once
+
 #define RGBLIGHT_EFFECT_BREATHE_TABLE
 
-const uint8_t rgblight_effect_breathe_table[] PROGMEM = {
-/* #define RGBLIGHT_EFFECT_BREATHE_CENTER   1.85 */
-/* #define RGBLIGHT_EFFECT_BREATHE_MAX      255 */
+// clang-format off
+
+// Breathing center: 1.85
+// Breathing max:    255
 
+const uint8_t PROGMEM rgblight_effect_breathe_table[] = {
```

Comment:
To be honest, I'm not sure where to put the `PROGMEM`.
However, the sample in the gcc manual is placed like this.

```suggestion
const uint8_t rgblight_effect_breathe_table[] PROGMEM = {
```

https://gcc.gnu.org/onlinedocs/gcc/AVR-Variable-Attributes.html

Reply:
AFAICT, it doesn't matter - keymaps have it between the type and identifier.